src/services/variableGetters.ts

```diff
@@ -0,0 +1,10 @@
+import { CustomVariable, sceneGraph, SceneObject } from '@grafana/scenes';
+import { VAR_LOGS_FORMAT } from './variables';
+
+export function getLogsFormatVariable(sceneRef: SceneObject) {
```

Comment:
we've got a bunch of variable getters spread around, should we collect them all somewhere?

Reply:
Yeah, we could use a clean up. Today I ran this quick search https://github.com/search?q=repo%3Agrafana%2Fexplore-logs%20sceneGraph.lookupVariable&type=code and, honestly, expected it to be worse.

Reply:
Yeah, we could use a clean up. Today I ran this quick search https://github.com/search?q=repo%3Agrafana%2Fexplore-logs%20sceneGraph.lookupVariable&type=code and, honestly, expected it to be worse.

Reply:
https://github.com/grafana/explore-logs/pull/632